cli/setup.py

```diff
@@ -14,11 +14,12 @@
     url='https://mozilla.github.io/fathom/',
     install_requires=['click>=7.0,<8.0', 'tensorboardX>=1.6,<2.0', 'torch>=1.0,<2.0'],
     entry_points={'console_scripts': [
+        'fathom-extract = fathom_web.extract:main'
```

Comment:
I think we should make these alphabetical

Reply:
Yes please!

Reply:
Also, the missing comma at the end of the line here wreaks havoc. :-)

Reply:
Darn, I don't know how I missed this. Thanks for catching it!